deploy/charts/harvester/values.yaml

```diff
@@ -377,3 +387,8 @@ webhook:
     tag: master-head
 
   httpsPort: 9443
+
+kube-vip:
+  enabled: true
+  config:
+      vip_interface: eth0
```

Comment:
Does this mean VIP is always on `eth0`?

Reply:
It's the default value. Once users enable the VLAN network and use eth0 as the VLAN NIC, eth0 is unavailable for the VIP interface and we will switch the VIP interface as `harvester-br0`.

Reply:
I don't see we override it in installation. Should we set it to be the management NIC?